news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title: osu!monthly #1
+date: 2022-07-28
+author: Soetch, Blushing, wowcake, 0x84f
+---
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)
+
+# Introducing osu!monthly
+
+It has been a long time since the last osu!weekly was released and luckily, we have something exciting to show you: osu!monthly! The new community news post format.
+
+6 years passed in which the community didn't have a regular post. Today, we're coming in with ***osu!monthly***, a bigger format.
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)
```

Comment:
This image is now there two times. Not sure if the position in the article makes a difference.

Reply:
Fixed.